src/components/DocumentationTopic.vue

```diff
@@ -615,7 +615,7 @@ export default {
         identifier: {
           interfaceLanguage,
           url: identifier,
-        },
+        } = {},
```

Comment:
Was there a specific need for this change? Off the top of my head, I can't think of a scenario where we ever try to render a page without an identifier.